--- src/AppRun.orig	2019-12-07 22:33:00.211967930 -0600
+++ ../src/AppRun.c	2019-12-07 22:44:55.739721814 -0600
@@ -155,7 +155,7 @@
     // change directory
     size_t appdir_s = strlen(appdir);
     char *usr_in_appdir = malloc(appdir_s + 5);
-    snprintf(usr_in_appdir, appdir_s + 5, "%s/usr", appdir);
+    snprintf(usr_in_appdir, appdir_s + 5, "%s/", appdir);
     ret = chdir(usr_in_appdir);
     if (ret != 0)
         die("Could not cd into %s\n", usr_in_appdir);
@@ -165,34 +165,6 @@
     size_t length;
     const char *format;
 
-    /* https://docs.python.org/2/using/cmdline.html#envvar-PYTHONHOME */
-    SET_NEW_ENV(new_pythonhome, appdir_s, "PYTHONHOME=%s/usr/", appdir);
-
-    old_env = getenv("PATH") ?: "";
-    SET_NEW_ENV(new_path, appdir_s*5 + strlen(old_env), "PATH=%s/usr/bin/:%s/usr/sbin/:%s/usr/games/:%s/bin/:%s/sbin/:%s", appdir, appdir, appdir, appdir, appdir, old_env);
-
-    old_env = getenv("LD_LIBRARY_PATH") ?: "";
-    SET_NEW_ENV(new_ld_library_path, appdir_s*10 + strlen(old_env), "LD_LIBRARY_PATH=%s/usr/lib/:%s/usr/lib/i386-linux-gnu/:%s/usr/lib/x86_64-linux-gnu/:%s/usr/lib32/:%s/usr/lib64/:%s/lib/:%s/lib/i386-linux-gnu/:%s/lib/x86_64-linux-gnu/:%s/lib32/:%s/lib64/:%s", appdir, appdir, appdir, appdir, appdir, appdir, appdir, appdir, appdir, appdir, old_env);
-
-    old_env = getenv("PYTHONPATH") ?: "";
-    SET_NEW_ENV(new_pythonpath, appdir_s + strlen(old_env), "PYTHONPATH=%s/usr/share/pyshared/:%s", appdir, old_env);
-
-    old_env = getenv("XDG_DATA_DIRS") ?: "/usr/local/share/:/usr/share/";
-    SET_NEW_ENV(new_xdg_data_dirs, appdir_s + strlen(old_env), "XDG_DATA_DIRS=%s/usr/share/:%s", appdir, old_env);
-
-    old_env = getenv("PERLLIB") ?: "";
-    SET_NEW_ENV(new_perllib, appdir_s*2 + strlen(old_env), "PERLLIB=%s/usr/share/perl5/:%s/usr/lib/perl5/:%s", appdir, appdir, old_env);
-
-    /* http://askubuntu.com/questions/251712/how-can-i-install-a-gsettings-schema-without-root-privileges */
-    old_env = getenv("GSETTINGS_SCHEMA_DIR") ?: "";
-    SET_NEW_ENV(new_gsettings_schema_dir, appdir_s + strlen(old_env), "GSETTINGS_SCHEMA_DIR=%s/usr/share/glib-2.0/schemas/:%s", appdir, old_env);
-
-    old_env = getenv("QT_PLUGIN_PATH") ?: "";
-    SET_NEW_ENV(new_qt_plugin_path, appdir_s*10 + strlen(old_env), "QT_PLUGIN_PATH=%s/usr/lib/qt4/plugins/:%s/usr/lib/i386-linux-gnu/qt4/plugins/:%s/usr/lib/x86_64-linux-gnu/qt4/plugins/:%s/usr/lib32/qt4/plugins/:%s/usr/lib64/qt4/plugins/:%s/usr/lib/qt5/plugins/:%s/usr/lib/i386-linux-gnu/qt5/plugins/:%s/usr/lib/x86_64-linux-gnu/qt5/plugins/:%s/usr/lib32/qt5/plugins/:%s/usr/lib64/qt5/plugins/:%s", appdir, appdir, appdir, appdir, appdir, appdir, appdir, appdir, appdir, appdir, old_env);
-
-    /* Otherwise may get errors because Python cannot write __pycache__ bytecode cache */
-    putenv("PYTHONDONTWRITEBYTECODE=1");
-
     /* Run */
     ret = execvp(exe, outargptrs);
     
@@ -203,14 +175,5 @@
 
     free(line);
     free(desktop_file);
-    free(usr_in_appdir);
-    free(new_pythonhome);
-    free(new_path);
-    free(new_ld_library_path);
-    free(new_pythonpath);
-    free(new_xdg_data_dirs);
-    free(new_perllib);
-    free(new_gsettings_schema_dir);
-    free(new_qt_plugin_path);
     return 0;
 }
